.gitignore

```diff
@@ -14,3 +14,4 @@ venv
 /src/iris/ui/static/bundles
 .tox
 *.orig
+logs/sender_rpc.access.log
```

Comment:
What is this file?

Reply:
iris-sender will likely write that rpc log. We should probably put changes to the .gitignore file in its own PR.

Reply:
removed